source/GameLoadingPanel.cpp

```diff
@@ -62,9 +63,17 @@ void GameLoadingPanel::Step()
 		player.LoadRecent();
 
 		GetUI()->Pop(this);
-		GetUI()->Push(new MenuPanel(player, gamePanels));
+		if(conversation.IsEmpty())
+			GetUI()->Push(new MenuPanel(player, gamePanels));
 		GetUI()->Push(new MenuAnimationPanel());
```

Comment:
What about the menu animation panel? Probably should be not shown either.

Reply:
I think it's helpful to have sounds when it is loading.  In the background you get a ding when the conversation pops up.

Reply:
Sounds help with player accessibility.  For when it's loading and the player is focused on something else.  It let's the player know the conversation is ready to look at.